test/provisioner/libvirt/provision_common.go

```diff
@@ -14,17 +12,13 @@ import (
 	"path"
 	"path/filepath"
 
+	pv "github.com/confidential-containers/cloud-api-adaptor/test/provisioner"
 	log "github.com/sirupsen/logrus"
 	"libvirt.org/go/libvirt"
 	"libvirt.org/go/libvirtxml"
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
-func init() {
-	newProvisionerFunctions["libvirt"] = NewLibvirtProvisioner
-	newInstallOverlayFunctions["libvirt"] = NewLibvirtInstallOverlay
-}
-
 // LibvirtProvisioner implements the CloudProvisioner interface for Libvirt.
 type LibvirtProvisioner struct {
 	conn         *libvirt.Connect // Libvirt connection
```

Comment:
Hey DaLi, I'm hitting error here:
```
# github.com/confidential-containers/cloud-api-adaptor/test/provisioner/libvirt
../provisioner/libvirt/provision_common.go:24:24: undefined: libvirt.Connect
../provisioner/libvirt/provision_common.go:73:23: undefined: libvirt.NewConnect
../provisioner/libvirt/provision_common.go:129:18: undefined: libvirt.StoragePool
../provisioner/libvirt/provision_common.go:163:54: undefined: libvirt.STORAGE_VOL_CREATE_PREALLOC_METADATA
../provisioner/libvirt/provision_common.go:224:59: undefined: libvirt.STORAGE_VOL_UPLOAD_SPARSE_STREAM
../provisioner/libvirt/provision_common.go:234:33: undefined: libvirt.Stream
../provisioner/libvirt/provision_common.go:265:57: undefined: libvirt.StoragePool
```
I think that now that this file is in the `libvirt` package that is shadowing things that were referenced from the `libvirt.org/go/libvirt` import, so we might need to considering aliasing that import?

Reply:
> ../provisioner/libvirt/provision_common.go:24:24: undefined: libvirt.Connect

@stevenhorsman Thanks for those review comments.
What's your dev machine system? Please have a try on linux machine, it seems that import libvirt package didn't work well on MAC os.

Reply:
I'm running this on a ubuntu 22.04 system

Reply:
Also, sorry I trimmed the console log too much earlier, I saw this error when running: `root@sh-ibmcloud-sm-pp-test1:~/go/src/github.com/confidential-containers/cloud-api-adaptor/test/tools# make BUILTIN_CLOUD_PROVIDERS="ibmcloud" all`

Reply:
fixed.

Reply:
Sorry, I forgot to add yesterday evening that this is now working for me. Thanks!